packages/graphile-build-pg/src/plugins/PgMutationCreatePlugin.js

```diff
@@ -44,6 +57,16 @@ export default (function PgMutationCreatePlugin(
           .filter(table => !!table.namespace)
           .filter(table => table.isSelectable)
           .filter(table => table.isInsertable)
+          .filter(
+            table =>
+              pgEnableDefaultMutationTables.length == 0 ||
+              pgEnableDefaultMutationTables.some(function(element) {
+                return (
+                  element.namespaceName == table.namespace.name &&
+                  element.typeName == table.name
```

Comment:
We should change `parseTypeIdentifier` into simply `parseIndentifier` and move it into a common library file. We should also change `typeName` to `name` along with this change.